L1Trigger/L1TGlobal/plugins/TriggerMenuParser.cc

```diff
@@ -2695,6 +2706,97 @@ bool l1t::TriggerMenuParser::parseEnergySumCorr(const L1TUtmObject* corrESum, un
   return true;
 }
 
+/**                                                                                                                                                            
+ * parseEnergySumCorr Parse an "energy sum" correlation condition and insert an entry to the conditions map                                                    
+ *                                                                                                                                                             
+ * @param node The corresponding node.                                                                                                                        
+ * @param name The name of the condition.                                                                                                                      
+ * @param chipNr The number of the chip this condition is located.                                                                                             
+ *                                                                                                                                                             
+ * @return "true" if succeeded, "false" if an error occurred.                                                                                                  
+ *                                                                                                                                                             
+ */
+
+bool l1t::TriggerMenuParser::parseAXOL1TL(L1TUtmCondition condAXOL1TL, unsigned int chipNr) {
+  using namespace tmeventsetup;
+
+  // get condition, particle name and particle type
+  std::string condition = "axol1tl";
+  std::string type = l1t2string(condAXOL1TL.getType());
+  std::string name = l1t2string(condAXOL1TL.getName());
+
+  LogDebug("TriggerMenuParser") << " ****************************************** " << std::endl
+                                << "     (in parseAXOL1TL) " << std::endl
+                                << " condition = " << condition
+                                << std::endl
+                                // << " particle  = " << particle << std::endl
+                                << " type      = " << type << std::endl
+                                << " name      = " << name << std::endl;
+
+  int nrObj = 1;
```

Comment:
`nrObj` seems to be treated as a constant throughout `parseAXOL1TL`. This can be reduced down to a `const` or `constexpr`